packages/components/src/tooltip/test/index.tsx

```diff
@@ -310,4 +310,26 @@ describe( 'Tooltip', () => {
 			await screen.findByRole( 'button', { description: 'tooltip text' } )
 		).toBeInTheDocument();
 	} );
+
+	it( 'should not hide tooltip when the anchor is clicked if hideOnClick is false', async () => {
```

Comment:
Might be useful to have a test to confirm that if `hideOnClick` is `true` or not specified, the tooltip still gets hidden on anchor click.

Reply:
We have a test that is kind-of implicitly testing for that

https://github.com/WordPress/gutenberg/blob/bfe85500a00e7e78f42117c6ded62e583bbc904e/packages/components/src/tooltip/test/index.tsx#L85-L97

Although I'd understand if you preferred having a more explicit check.

Reply:
I guess we can also expand the test description if we want, but yeah, that test does the job IMHO 👍